go/ql/test/query-tests/InconsistentCode/DeferredErrorCall/tests.go

```diff
@@ -0,0 +1,23 @@
+package test
+
+import (
+	"errors"
+)
+
+func returnsPair() (string, error) {
+	return "", errors.New("Test error")
+}
+
+func checkSomething() error {
+	return nil
+}
+
+func deferredCalls() {
+	defer returnsPair()
+	defer checkSomething()
```

Comment:
If you want to be thorough, you can cover some more cases: (1) assign a function to a variable and use the variable in the defer statement; (2) defer a function literal.

Reply:
Added in https://github.com/github/codeql/pull/11410/commits/5620d247fd0d1763fdf8fe62bb3683481bc4ea25